src/main/kotlin/uk/gov/justice/digital/hmpps/hmppshdcapi/licences/LicenceService.kt

```diff
@@ -56,4 +64,86 @@ class LicenceService(
       addressTown = addressObject.addressTown,
       postCode = addressObject.postCode,
     )
+
+  private fun formatCurfewHoursObject(curfewHours: CurfewHours, licenceId: Long?): List<CurfewTimes> {
+    val day1 =
+      CurfewTimes(
+        licenceId = licenceId,
+        fromDay = "Monday",
+        fromTime = LocalTime.parse(curfewHours.mondayFrom),
+        untilDay = "Tuesday",
+        untilTime = LocalTime.parse(curfewHours.tuesdayUntil),
+        curfewTimesSequence = 1,
+        createdTimestamp = LocalDateTime.now(clock),
+      )
+    val day2 =
+      CurfewTimes(
+        licenceId = licenceId,
+        fromDay = "Tuesday",
+        fromTime = LocalTime.parse(curfewHours.tuesdayFrom),
+        untilDay = "Wednesday",
+        untilTime = LocalTime.parse(curfewHours.wednesdayUntil),
+        curfewTimesSequence = 2,
```

Comment:
**niggle:** You could possibly also get rid of the sequence as well? 
Arrays are ordered so the sequence is implicit.. this could also be seen as an implementation detail of how the data is stored in CVL